plugins/woocommerce/tests/php/src/Blocks/BlockTypes/ProductCollection/SqlGeneration.php

```diff
@@ -0,0 +1,541 @@
+<?php
+
+namespace Automattic\WooCommerce\Tests\Blocks\BlockTypes\ProductCollection;
+
+use Automattic\WooCommerce\Tests\Blocks\Mocks\ProductCollectionMock;
+use WC_Helper_Product;
+use WC_Tax;
+use WP_Query;
+use Automattic\WooCommerce\Enums\ProductStockStatus;
```

Comment:
Super nitpick: I know this is the order from original file but what about better ordering?

```suggestion
use Automattic\WooCommerce\Tests\Blocks\Mocks\ProductCollectionMock;
use Automattic\WooCommerce\Enums\ProductStockStatus;
use WC_Helper_Product;
use WC_Tax;
use WP_Query;
```